firmware/lm32/heartbeat.h

```diff
@@ -0,0 +1,21 @@
+#ifndef __HEARTBEAT_H
+#define __HEARTBEAT_H
+
+#define HDMI_IN0_FRAMEBUFFERS_BASE 	0x01000000
```

Comment:
Moved FRAMEBUFFER_BASE definitions to respective header files. Although I not using these definitions now, I am directly using the hdmi_inx_framebuffer_base() function.